benchmark/index.js

```diff
@@ -661,8 +695,6 @@ document.addEventListener('DOMContentLoaded', () => {
           let preferOpt = preferSelectElement.options[i];
           if (preferOpt.value === 'sustained') {
             preferOpt.selected = true;
-          } else if (preferOpt.value === 'fast') {
-            preferOpt.disabled = true;
           } else if (preferOpt.value === 'low') {
```

Comment:
@BruceDai any reason to remove fast here?

Reply:
@ibelem This removed code is for showing disabled FAST_SINGLE_ANSWER preference item when testing on Windows and Linux, now recent implemented mklDNN backend uses FAST_SINGLE_ANSWER preference, so here needs to enable FAST_SINGLE_ANSWER preference item, so removed this code, thanks